kura/features/pom.xml

```diff
@@ -8,7 +8,7 @@
 		<groupId>org.eclipse.kura</groupId>
 		<artifactId>kura</artifactId>
 		<version>3.1.0-SNAPSHOT</version>
-		<relativePath>../manifest_pom.xml</relativePath>
+		<relativePath>..</relativePath>
```

Comment:
You can remove <relativePath> because parent pom is now in the correct location. Here and in other pom.xml files.